filebeat/config/config.go

```diff
@@ -44,32 +44,32 @@ type FilebeatConfig struct {
 }
 
 type ProspectorConfig struct {
-	Paths                 []string
+	ExcludeFiles          []string `yaml:"exclude_files"`
+	ExcludeFilesRegexp    []*regexp.Regexp
```

Comment:
I don't think so as this is the object generated out of ExcludeFiles which is the config entry